### PR TITLE
Prepared role for Ansible Galaxy

### PR DESCRIPTION
**Pull request by turnrye**
These changes required restructuring the repo a bit, but the meat of it is the same (tasks, templates). I've also moved some vars into defaults and left others to be hostvars. Things like API key, ID, lat/lon are all host-specific, so they make sense to be there instead of as part of the group.

I've not added this to ansible galaxy because I didn't want it tied to my personal account. Also, I've not setup testing yet -- I figured that'd be a good next step for a separate PR. The testing that's currently present is from the ansible galaxy init method and can safely be overlooked as "default".

### CODE REVIEW

**Comment by turnrye** [comment]
I've also added an easy install script that can be run on the local rpi for stupid simple installation.

**Comment by turnrye** [comment]
Added a task to auto-run Kalibrate on the local device now

**Comment by johngian** [comment]
Please remove `?` from the commit message here:
https://github.com/satnogs/satnogs-client-ansible/pull/1/commits/ad31dfb835a76d5ff4ff2a20cc00ecddb35ea3b8

File: README.md [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+None!
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: satnogs-client-ansible
+      roles:
+         - { role: ryan_turner.satnogs-client-ansible, satnogs_api_url: https://network.satnogs.org/api/, satnogs_api_token: 123, satnogs_station_id: 456, satnogs_station_lat: 35.0, satnogs_station_lon: -89, satnogs_station_elev: 89, satnogs_ppm_error: -2 }
```
Comment (johngian) [inline README.md]: The role here should be changed to `satnogs.satnogs-client-ansible`.

Comment (johngian) [inline README.md]: This means that we should also create an account and register our role to galaxy.

File: README.md [hunk 2]
```diff
@@ -21,11 +31,16 @@ None!
...
 
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
-
-    - hosts: satnogs-client-ansible
-      roles:
-         - { role: ryan_turner.satnogs-client-ansible, satnogs_api_url: https://network.satnogs.org/api/, satnogs_api_token: 123, satnogs_station_id: 456, satnogs_station_lat: 35.0, satnogs_station_lon: -89, satnogs_station_elev: 89, satnogs_ppm_error: -2 }
+```
+- hosts: satnogs
+  roles:
+    - { role: ryan_turner.satnogs-client-ansible}
```
Comment (johngian) [inline README.md]: Same here.

File: easy-install.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
+sudo apt-get update
+sudo apt-get install python3-pip git -y
```
Comment (johngian) [inline easy-install.sh]: Why do you explicitly install `python3-pip` ?
FYI satnogs-client is mostly written and tested in python 2.7

**Comment by johngian** [comment]
Please squash the README update commits.

**Comment by johngian** [comment]
No need to have merge commits in this PR. Please squash.

**Comment by johngian** [comment]
Code looks OK. Please update the PR after the comments and then we can merge.

**Comment by turnrye** [comment]
"This means that we should also create an account and register our role to galaxy."

Willl you take care of that, @johngian? I'm happy to do it but I still have no formal affiliation with SatNOGS.

**Comment by turnrye** [comment]
I squashed as you liked, but accidentally went one too far... trying to figure out how to undo that now.

**Comment by johngian** [comment]
Hi @ryanturner. Is there any update with this PR? It looks like it has merge conflicts with master.

**Comment by turnrye** [comment]
That's all due to a goofed up squash... I may end up just re-doing these changes in a new PR, because I haven't figured out how to undo that mistake.

**Pull request closed without merging**